src/views/studio/VideoWorkspace/VideoWorkspaceForm/VideoWorkspaceForm.style.ts

```diff
@@ -37,7 +37,7 @@ type FormScrollingProps = {
 }
 export const FormScrolling = styled.div<FormScrollingProps>`
   height: ${({ actionBarHeight, isEdit }) =>
-    isEdit ? '100%' : `calc(100% - ${actionBarHeight ?? 0}px - ${EDIT_VIDEO_TABS_BAR_HEIGHT}px)`};
+    isEdit ? '100%' : `calc(100% - ${actionBarHeight ?? 0}px - ${EDIT_VIDEO_TABS_BAR_HEIGHT}px) `};
```

Comment:
Unnecessary space added here I think?

Reply:
Hmm, 🤔  Actually Im getting warning from stylelint: 
![image](https://user-images.githubusercontent.com/51168865/135459953-f3415a3f-c148-42ef-bea5-8043e2f3608e.png)

Reply:
hmm, stylelint is not really working that great atm :P but whatever